src/node.cc

```diff
@@ -2843,6 +2844,11 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_print_eval", True(env->isolate()));
   }
 
+  // -c, --check
+  if (syntax_check_only) {
+    READONLY_PROPERTY(process, "_syntax_check_only", True(env->isolate()));
```

Comment:
Is this needed? the syntax check won't actually launch the script so there will never be access to `process._syntax_check_only`.